apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -203,7 +204,7 @@ export const ProgressDot = Radium(React.createClass({
         ((level.ids && level.ids.map(id => id.toString()).indexOf(this.props.currentLevelId) !== -1) ||
         level.uid === this.props.currentLevelId);
 
-    const isUnplugged = isNaN(level.title);
+    const isUnplugged = level.kind === LevelKind.unplugged;
```

Comment:
level.title comes from https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script_level.rb#L162. Level.kind (which comes from https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script_level.rb#L181) should be a more clear way of expressing the exact same thing.